scout/server/blueprints/mme/templates/mme/mme_submissions.html

```diff
@@ -0,0 +1,128 @@
+{% extends "layout.html" %}
+{% from "overview/institute_sidebar.html" import institute_actionbar %}
+
+{% block title %}
+{{ super() }} - Institutes
+{% endblock %}
+
+{% block css %}
+{{ super() }}
+<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/dataTables.bootstrap5.min.css">
+{% endblock %}
+
+{% block top_nav %}
+  {{ super() }}
+  <li class="nav-item">
+    <a class="nav-link" href="{{ url_for('cases.index') }}">Institutes</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" href="{{ url_for('overview.cases', institute_id=institute._id) }}">{{ institute.display_name }}</a>
+  </li>
+  <li class="nav-item active d-flex align-items-center">
+    <span class="navbar-text">Cases in MatchMaker Exchange</span>
+  </li>
+{% endblock %}
+
+{% block content_main %}
+<div class="container-float">
+  <div class="row mt-3" id="body-row"> <!--sidebar and main container are on the same row-->
+    <div class="col">
+      {{ mme_submitted() }}
+    </div>
+  </div>
+</div>
+{% endblock %}
+
+{% macro mme_submitted() %}
+<table class="table table-bordered table-striped" id="mmeTable">
+  <thead>
+    <tr>
+      <th>Case Name</th>
+      <th>Synopsis</th>
+      <th>Status</th>
+      <th>Case Assignees</th>
+      <th>MME Submission Updated</th>
+      <th>MME Patient(s)</th>
+      <th>MME Phenotype Features</th>
+      <th>MME Genomic Features</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% for case in mme_cases %}
+      <tr>
+        <td>
+          <a class="me-2" href="{{ url_for('cases.case', institute_id=case.owner, case_name=case.display_name) }}" target="_blank" rel="noopener">
+          {{ case.display_name }}
+          </a>
+        </td>
+        <td>{{ case.synopsis }}</td>
+        <td>{{ case.status }}</td>
+        <td>
+          {% for assignee in case.assignees %}
+            <div>{{ assignee }}</div>
+          {% endfor %}
+        </td>
+        <td>
+          {{ case.mme_submission.updated_at.strftime('%Y-%m-%d %H:%M') }}
+        </td>
+        <td>
+          {% for patient in case.mme_submission.patients %}
+            <div>
+              <strong>{{ patient.label }}</strong> ({{ patient.sex }})<br>
+              <small>Contact: {{ patient.contact.name }} - <a href="{{ patient.contact.href }}">{{ patient.contact.href }}</a></small>
+            </div>
+          {% endfor %}
+        </td>
+        <td>
+          {% for disorder in case.mme_submission.disorders %}
+            <div>{{ disorder.label }} ({{ disorder.id }})</div>
+          {% endfor %}
+        </td>
+        <td>
+          {% for patient in case.mme_submission.patients %}
+            {% for gf in patient.genomicFeatures %}
+              <div>
+                <strong>{{ gf.gene.id }}</strong><br>
+                {% if gf.variant %}
+                  Chr{{ gf.variant.referenceName }}:
+                  {{ gf.variant.start }} → {{ gf.variant.end }}<br>
```

Comment:
```suggestion
                  {{ gf.variant.start|human_longint|safe }}-{{ gf.variant.end|human_longint|safe }}<br>
```
add thin space mangling.. :)